src/content/docs/crowdin/translation-process/crowdin-ai.mdx

```diff
@@ -204,22 +207,58 @@ To disable the AI provider, follow these steps:
   2. In the **Providers** tab, click <Icon name="mdi:dots-horizontal" class="inline-icon" /> toward the needed provider and select **Disable**. Alternatively, double-click on the needed provider and clear **Enabled** in the provider's settings.
 </Steps>
 
-### Topping Up Account Balance
+### Adding Funds to Account Balance
 
-MT engines and AI models managed by Crowdin are separately paid services. To use them, you first need to top up your Crowdin account balance.
+MT engines and AI models managed by Crowdin are separately paid services. To use them, you first need to add funds to your Crowdin account balance.
 
 <ReadMore>
-  Read more about [Topping Up Account Balance](/crowdin-managed-services/#topping-up-account-balance).
+  Read more about [Adding Funds to Account Balance](/crowdin-managed-services/#adding-funds-to-account-balance).
 </ReadMore>
 
 ### AI Settings for the Account
 
-To configure default settings for AI prompts across your Crowdin account, open your profile and go to **AI > Settings**.
+To configure default AI settings across your Crowdin account, open your profile and go to **AI > Settings**.
 
-**Prompt to be used in editor** &ndash; select the default AI prompt that will be used in the Editor for all projects in your Crowdin account. If needed, this default setting can be overridden for each project individually in the project's **Settings > AI > Settings**.
+These default settings can be overridden for each project individually in the project's **Settings > AI > Settings** as needed.
 
 <Image src={aiSettings} alt="AI Settings" />
 
+* **AI Assistant in editor** &ndash; select the default prompt for interactions with AI Assistant in the Editor across all projects in your Crowdin account.
+
+* **AI Suggestion in editor** &ndash; select the default prompt for AI suggestions that will appear in the Editor across all projects in your Crowdin account. If no account-wide setting is configured and no prompt is manually selected for a specific project, AI suggestions will not be displayed in the Editor for that project. Once a prompt is selected, Crowdin will use it to generate and display translation suggestions in the Editor.
+
+<Aside>
+  Using **AI Assistant in editor** and **AI Suggestion in editor** may increase token usage.
+</Aside>
+
+* **Shortcuts for chat with AI in editor** &ndash; shortcuts provide quick actions in the Editor to start AI interactions. Create shortcuts with specific instructions for the AI to quickly access frequently used actions across projects. These shortcuts are global, applying to all projects, although individual users can create their own in the **Editor Settings > AI** tab.
```

Comment:
Create separate headings for these sections (AI Assistant in editor, AI Suggestion in editor, Shortcuts for chat with AI in editor, etc.) instead of the list. It's more preferable for such cases when the item has a lot of content in the description.

Reply:
Reformatted with separate headings